ref/feature/mpJwt/examples.adoc

```diff
@@ -13,23 +15,40 @@ The `<mpJwt/>` element is required to configure the processing of the caller’s
         <feature>cdi-2.0</feature>
         <feature>appSecurity-3.0</feature>
     </featureManager>
+----
 
+This is the keystore that is used by SSL and by JWT.
+The keystore is built by using the maven keytool plugin,
 
-    <!-- This is the keystore that will be used by SSL and by JWT.
-         The keystore is built using the maven keytool plugin -->
     <keyStore id="defaultKeyStore" location="keystore.jceks" type="JCEKS"
               password="secret" />
 
-    <!-- The HTTP ports that the application will use. -->
+These are the HTTP ports that the application uses,
```

Comment:
don't use "These" as the subject- could be rewritten in "The following example shows..." format